### ISSUE TITLE
Re-train few epochs more

### ISSUE THREAD

**Issue by JordinaAviles** [MIC-DKFZ/nnUNet#270]
Hi, 

Thank you for the amazing job done! It's really intuitive and helpful!

I have a doubt regarding the use of pre-trained models. I already trained a 3d_full res and now I wanted to assess its performance with a completely unseen dataset with images from a different scan. I expect the accuracy to drop so my idea is to put some of the images from the new dataset in the already trained model and train for some epochs more. How can I do that? This means, picking up my trained model with dataset 1 and put in the training cases some of the data in dataset2 but with the weights already obtained (not random as in the beggining) and train for just some epochs more.

Thank you,

Jordina

**Comment by FabianIsensee** [comment]
You can simply create a new task with the additional training data, load the checkpoint prior to training and then train. This may need some adaptation wrt momentum, learning rate and numbere of epochs though

**Comment by Joevaen** [comment]
> This may need some adaptation wrt momentum, learning rate and numbere of epochs though

Hi, fabian!
` This may need some adaptation wrt momentum, learning rate and numbere of epochs though` what does this mean? Just reload former model on a new dataset doesn't work?
